src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,34 @@
+@use "../utils/variables";
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    padding: 22.5px 0;
```

Comment:
I'm sceptical about this .5px here, try to style this link with a height property
As a rule: padding for buttons, height for regular elements